cluster/gce/gci/configure-helper.sh

```diff
@@ -2156,11 +2156,15 @@ EOF
        [[ "${METADATA_AGENT_VERSION:-}" != "" ]]; then
       metadata_agent_cpu_request="${METADATA_AGENT_CPU_REQUEST:-40m}"
       metadata_agent_memory_request="${METADATA_AGENT_MEMORY_REQUEST:-50Mi}"
+      metadata_agent_cluster_level_cpu_request="${METADATA_AGENT_CLUSTER_LEVEL_CPU_REQUEST:-40m}"
```

Comment:
@supriyagarg @x13n 
Do we have any more up-to-date numbers, so that we can set reasonable defaults?

Reply:
I don't have any up-to-date numbers. Let me check with others in the agents team.

Reply:
Let's not block this PR, but set reasonable defaults separately

Reply:
This is way too high to be acceptable (but the same goes for the DaemonSet one).